book/src/feature/wasm.md

```diff
@@ -30,3 +30,8 @@ flag as well.
 
 Check out [Integrating with Web](../integrate/web.md) for instructions
 on how to consume the web bridge.
+
+What's more, you don't have to use --dart-decl-output,
```

Comment:
There's a line above that shows the `wasm` flag together with `dart-decl-output`, could you please adjust it to make it consistent? Then this line shouldn't be needed.

Reply:
I can't understand :/ Can you give me an example?
... -_-||

Reply:
Or just modify it